test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Passive mode is great when you have an easy and clear way to trigger authentication (like the `HostAuthenticationFilter/Attribute` in Web API OWIN). Currently, the only OTB solution is through the authorization stuff.

Making it manual by default is probably going to confuse users, but that's just my 2 cents.